python/lsst/meas/base/applyApCorr.py

```diff
@@ -105,6 +126,11 @@ class ApplyApCorrConfig(lsst.pex.config.Config):
         dtype=bool,
         default=True,
     )
+    prefixList = lsst.pex.config.ListField(
+        doc="Prefixes to search for to find additional fields to be aperture-corrected",
+        dtype=str,
+        default=["undeblended_",]
+    )
```

Comment:
I'm not sure I like conflating "having a prefix" with "uses another flux to define its aperture corrections".

Could we make this a `DictField` that maps "to-be-corrected" flux to "flux-to-use-for-correction"?  Or better yet, have algorithms specify the flux that should be used to correct them when they're registered?

Side note: this discussion gives me a slight feeling of deja vu, like we already discussed this.  If we did, and we already agreed on the approach you have here, just tell me (though I'd appreciate a reminder of how we got here, or a link to that discussion).  It's also quite possible I just had a conversation about a similar configuration option with someone else.